tests/examples_workinprogress/workinprogress_modelbuilder.py

```diff
@@ -14,7 +14,7 @@
 import xarray as xr
 import pandas as pd
 import contextily as ctx
-
+import warnings
```

Comment:
Minor thing, but please re-introduce the whitespace below the imports in this script